cmd/goal/clerk.go

```diff
@@ -79,6 +79,7 @@ var (
 	simulateStackChange           bool
 	simulateScratchChange         bool
 	simulateAllowUnnamedResources bool
+	simulateInitialStateEnable    bool
```

Comment:
The other flags don't say "Enable".  I was going to recommend `simulateInitialAppState` but I suppose we actually should be sending intial asset and box state, eventually, under the same flag (at least, I think we should use a single flag). So how about `simulateInitialResourceState`?

Reply:
right, I go with `simulateInitialResourceState` here.